### PR TITLE
PD-2020: Group 1.1

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes
**Dark Mode Updates**

🎟 _Jira ticket:_ [PD-2020](https://jira.mongodb.org/browse/PD-2020)
Typography: [PD-1940](https://jira.mongodb.org/browse/PD-1940)
Icon Button: [PD-2048](https://jira.mongodb.org/browse/PD-2048)
Text Area: [PD-2058](https://jira.mongodb.org/browse/PD-2058)
Tooltip: [PD-2071](https://jira.mongodb.org/browse/PD-2071)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: packages/tooltip/src/Tooltip.story.tsx [hunk 1]
```diff
@@ -20,8 +20,21 @@ export default {
...
+  darkMode,
+  ...args
+}: TooltipProps) => (
+  <div
+    className={css`
+      padding: 100px;
+    `}
+  >
+    <Tooltip
+      darkMode={darkMode}
```
Comment (spark33) [inline packages/tooltip/src/Tooltip.story.tsx:33]: Small thing, but it looks like `darkMode` doesn't have to be broken out of the spread any more.

File: packages/icon-button/src/IconButton.tsx [hunk 2]
```diff
@@ -152,52 +154,54 @@ const disabledStyle: Record<Mode, string> = {
 
     &:focus {
       color: ${palette.gray.light1};
+
       &:before {
-        background-color: ${palette.gray.light3};
+        background-color: rgba(255, 255, 255, 0);
```
Comment (TheSonOfThomp) [inline packages/icon-button/src/IconButton.tsx:159]: 👍

File: packages/typography/src/ChartLabel.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { baseTypographyStyles, defaultTextColor } from './styles';
+import { CommonTypographyProps, Mode } from './types';
+
+/**
+ * ChartLabel
+ */
```
Comment (spark33) [inline packages/typography/src/ChartLabel.tsx:9]: This TSDoc should be on the function definition instead. Could we also have some description of what this component is, and how it should be used?

File: packages/typography/src/Description.tsx [hunk 4]
```diff
@@ -20,7 +20,7 @@ const descriptionTypeScale: Record<BaseFontSize, string> = {
   `,
   [BaseFontSize.Body2]: css`
     font-size: ${typeScales.body2.fontSize}px;
-    line-height: ${typeScales.body2.lineHeight}px;
+    line-height: 20px;
```
Comment (spark33) [inline packages/typography/src/Description.tsx:23]: Why was this reverted to `20px`? It looks like `typeScales.body1.lineHeight`would work here

File: packages/typography/src/Label.tsx [hunk 5]
```diff
@@ -18,7 +18,7 @@ const labelTypeScale: Record<BaseFontSize, string> = {
   `,
   [BaseFontSize.Body2]: css`
     font-size: ${typeScales.body2.fontSize}px;
-    line-height: ${typeScales.body2.lineHeight}px;
+    line-height: 20px;
```
Comment (spark33) [inline packages/typography/src/Label.tsx:21]: Same as above - maybe this should be `typeScales.body1.lineHeight`

**Review by spark33 — APPROVED** [review]
Left some comments, but approving since they're all fairly minor!

**Review by TheSonOfThomp — APPROVED** [review]
Looks good 👍 Just a few minor details

File: packages/tooltip/src/Tooltip.story.tsx (hunk 1 above)
Comment (shaneeza) [reply packages/tooltip/src/Tooltip.story.tsx:33]: I did this so that i can pass darkMode to the trigger button

**Pull request merged**